core/src/test/java/org/elasticsearch/plugins/PluginManagerUnitTests.java

```diff
@@ -96,9 +96,9 @@ public void testSimplifiedNaming() throws IOException {
 
     @Test
     public void testTrimmingElasticsearchFromOfficialPluginName() throws IOException {
```

Comment:
Maybe renaming the test would be good since this is not testing that dropping the prefix works anymore.